juju-rrun

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+#
+# Copyright 2015 Felipe Reyes <felipe.reyes@canonical.com>
+#
+# juju-rrun is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# juju-clean is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# For a full copy of the GNU General Public License, see
+# <http://www.gnu.org/licenses/>.
+#
+# This plugin pretty prints stdout/stderr generated by 'juju run'
+#
+
+from __future__ import print_function
+import json
+import subprocess
+import sys
+
+separator = '----------'
+
+
+def print_help():
+    print('Pretty print of juju run output')
```

Comment:
You have these backwards. print_help should be what print_description has and vice versa. Description is oneline help is help output.